onnxruntime/core/graph/graph.cc

```diff
@@ -4084,82 +4086,104 @@ ONNX_NAMESPACE::GraphProto Graph::ToGraphProto() const {
   return result;
 }
 
-ONNX_NAMESPACE::GraphProto Graph::ToGraphProtoWithExternalInitializers(const std::filesystem::path& external_file_path,
-                                                                       const std::filesystem::path& model_file_path,
-                                                                       size_t initializer_size_threshold,
-                                                                       const OffsetAlignmentInfo& align_info) const {
-  GraphProto result;
-  ToGraphProtoInternal(result);
-  ORT_ENFORCE(external_file_path.is_relative());
-  // If model_file_path is just a file name without a path separator, for example: "model.onnx". Its parent path could
-  // be empty. Else, save external data file in same directory as the model.
-  const std::filesystem::path modified_external_file_path = model_file_path.parent_path() / external_file_path;
+// A recursive function that does bottom up with subgraphs
+Status Graph::ToGraphProtoWithExternalInitiallizersImpl(
```

Comment:
nit: this name might be confusing. IIUC the bulk of converting the the GraphProto is handled by the call to ToGraphProtoInternal. This implementation seems to be about updating the GraphProto with prepacked initializers once that GraphProto is created. 

i.e. it's more like 'add prepacked weights to graph proto' than being the overall implementation of 'create graph proto with external initializers'